hello.apex

```diff
@@ -1,13 +1,18 @@
-String flagKey = 'my-flag';
+# You will set your LaunchDarkly SDK key during the bridge.
+
+# Set flagKey to the feature flag key you want to evaluate.
+String flagKey = 'my-boolean-flag';
 
 LDConfig config = new LDConfig.Builder().build();
 
 LDClient client = new LDClient(config);
 
-LDUser user = new LDUser.Builder('abc').build();
+# Set up the user properties. This user should appear on your LaunchDarkly users
+# dashboard soon after you run the demo.
+LDUser user = new LDUser.Builder('example-user-key')
+    .setName('Sandy')
+    .build();
 
 if (client.boolVariation(user, flagKey, false)) {
-    System.debug('feature is on');
-} else {
-    System.debug('feature is off');
+    System.debug('Feature flag '+flagKey+' is '+value+' for this user');
```

Comment:
It doesn't look like `value` is defined here yet. Also, because you've updated the debug statement to print out the evaluated value, I think you want to print that value no matter what (whether it evaluates to true or false) -- so you don't need to wrap it in an `if` statement. What you currently have will only ever print anything if the flag evaluates to true. 

I think you want:

```
Boolean value = client.boolVariation(user, flagKey, false);
System.debug('Feature flag '+flagKey+' is '+value+' for this user');
```